<launch>

  <arg name="end_frame" default="/world"/>
  <arg name="min_confidence" default="0.7"/>
  <arg name="z_threshold" default="0.04"/>
  <!-- <arg name="output_frame" default="world"/> -->

  <node name="bb_to_world_node" pkg="bb_to_world" type="bb_to_world_node" args="$(arg end_frame)" output="screen">
    <remap from="b_box" to="tld_tracked_object"/>
    <remap from="image" to="/kinect2/sd/image_depth_rect" />
    <remap from="camera_info" to="/kinect2/sd/camera_info" />
    <param name="z_threshold" value="$(arg z_threshold)" />
    <param name="min_confidence" value="$(arg min_confidence)" />
  </node>

</launch>
